source/API/CMakeLists.txt

```diff
@@ -199,13 +202,15 @@ if(NOT LLDB_BUILT_STANDALONE)
 endif()
 
 # Copy the clang resource directory.
+if (LLDB_ENABLE_SWIFT_SUPPORT)
 add_custom_command_target(
     copy-clang-resource-dir
     COMMAND "${CMAKE_COMMAND}" "-E" "copy_directory" "${CLANG_RESOURCE_PATH}" "${lib_dir}/lldb/clang/"
     OUTPUT "${lib_dir}/lldb/clang/"
     VERBATIM
     ALL
     DEPENDS ${clang_headers_target})
+endif()
```

Comment:
More than just this needs to be guarded. Below is code to copy the swift resource dir as well, so that should be guarded too.